models/classes/class.QtiTestConverter.php

```diff
@@ -460,6 +460,19 @@ private function lookupClass($name)
             'qtism\\data\\rules\\',
             'qtism\\data\\state\\'
         ];
+
+        $operatorClassesPostfix = [
```

Comment:
The collection is better off defined either as a static variable or a private constant.